File: Assignment35_2.c
// Write a program which accepts string from user and count number of small characters.

#include<stdio.h>

int CountSmall(char *str)
{
    int iCount = 0;

    while(*str != '\0')
    {
        if((*str >= 'a') && (*str <= 'z'))
        {
            iCount++;            
        }
        str++;
    }
    return iCount;
}

int main()
{
    char Arr[] = {'\0'};
    int iRet = 0;

    printf("Enter a String: \n");
    scanf("%[^'\n']s",Arr);

    iRet = CountSmall(Arr);

    printf("Number of small characters in string are: %d\n",iRet);

    return 0;
}

File: Assignment35_2.java
// Write a program which accepts string from user and count number of small characters.
import java.util.Scanner;
class Logic
{
    public int CountSmall(String str)
    {
        char Arr[] = str.toCharArray();
        int iCount= 0;

        for(int i = 0; i < Arr.length; i++)
        {
            if((Arr[i] >= 'a') && (Arr[i] <= 'z'))
            {
                iCount++;
            }
        }
        return iCount;
    }
}

class Assignment35_2
{
    public static void main(String[] args) 
    {
        Scanner scanobj = new Scanner(System.in);

        System.out.println("Enter a string: ");
        String sobj = scanobj.nextLine();

        Logic lobj = new Logic();
        int iRet = lobj.CountSmall(sobj);    
        
        System.out.println("Number of small characters are: "+iRet);

    }
}